packages/client-js/src/manifest/generate.js

```diff
@@ -0,0 +1,17 @@
+const schema = require("@web3api/manifest-schema").default;
+const compile = require("json-schema-to-typescript").compile;
+const writeFileSync = require("fs").writeFileSync;
+
+const generateManifest = () => {
```

Comment:
Should this function be moved to the `manifest-schema` package? 🤔  Just thinking out loud here